src/renderer/services/neo.js

```diff
@@ -1049,6 +1049,22 @@ export default {
                 if (newAssetBalance) {
                   if (DBG_LOG) console.log(`Address '${forAddress}' Merging old balance for asset ${sym}`);
 
+                  const doMigration = (migrationType, utxo, blocksToDrop, migrationLambda) => {
+                    const migratedKey = `${migrationType}:${utxo.txid}:${utxo.index}`;
+                    let migratedBlock = existingBalance.migrated[migratedKey];
```

Comment:
I know I could have used _.get, _.set, and _.omit instead of the native operations; decided to just go with native object access for this though. Talk offline if you would like to convince me to use lodash ops for this.

Reply:
Seems fine to me.